services/QuillLMS/client/app/bundles/Teacher/components/classrooms/classroom_teacher_section.tsx

```diff
@@ -266,7 +267,7 @@ export default class ClassroomTeacherSection extends React.Component<ClassroomTe
     if (!classroom.visible) {
       return null
     } else {
-      return <button className="quill-button-archived primary outlined small" onClick={() => this.inviteCoteachers()}>Invite co-teachers</button>
+      return <button className="quill-button primary outlined small focus-on-light" onClick={() => this.inviteCoteachers()}>Invite co-teachers</button>
```

Comment:
I know this isn't part of what you changed, so feel free to ignore, but I think changing this line to `onClick={this.inviteCoteachers}` will have the same function and get rid of an optional linting error.

Reply:
will do!